config/discord.php

```diff
@@ -16,6 +16,9 @@
     'bot_secret' => getenv('DISCORD_BOT_SECRET'),
     'bot_token' => getenv('DISCORD_BOT_TOKEN'),
 
+    'webhook' => env('DISCORD_WEBHOOK', 'null'),
```

Comment:
Make webhook an array with a proper key so that in the future when we add more webhooks it doesn't become a mess :)